mesmerize_napari/cnmf_gui.py

```diff
@@ -38,7 +38,7 @@ def get_params(self, *args, group_params: bool = True) -> Tuple[str, bool, dict]
             "tsub": self.ui.spinBox_tsub.value(),
             "method_init": self.ui.comboBox_method_init.currentText(),
             # 'border_pix': bord_px,
-            # 'fr': self.vi.viewer.workEnv.imgdata.meta['fps']
+            'fr': self.ui.doubleSpinBoxFrameRate.value(),
```

Comment:
at the beginning of `get_params()` I would do a check for the framerate value and raise if it is `<= 0`